src/components/DocumentViewer/DocumentViewer.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Meta } from '@storybook/react';
+import DocumentViewer from './DocumentViewer';
+
+export default {
+	component: DocumentViewer,
+	title: 'DocumentViewer',
+} as Meta;
+
+export const PNGDocument = () => <DocumentViewer url="/assets/example.png" />;
```

Comment:
Have you considered decorating these with a Type and a storyName? I.e.:
```
export const SingleFile: Story = () => (
	<FileUpload
		{...defaultProps}
		options={{ noClick: true, noKeyboard: true, maxFiles: 1 }}
		placeholder="Sleep een bestand in dit vlak of"
	/>
);
SingleFile.storyName = 'Single file';
```

Reply:
Added type to story.